README.md

```diff
@@ -1,4 +1,4 @@
 # snd-hello-world
 this is my second hello world repository (this time, created by web interface)
 
-My first hello world has been created from shell, starting from a local folder previously created for that purpose.
+I created my first hello world using manual shell commands, starting from a local folder previously created just for that purpose.
```

Comment:
Could be better to use the third person?

Reply:
Marco, for me it's OK with the first person